pkg/kubelet/cm/devicemanager/device_plugin_stub.go

```diff
@@ -123,11 +124,18 @@ func (m *Stub) Start() error {
 		defer m.wg.Done()
 		m.server.Serve(sock)
 	}()
-	_, conn, err := dial(m.socket)
-	if err != nil {
+
+	if err := wait.PollImmediate(1*time.Second, 10*time.Second, func() (bool, error) {
+		_, conn, err := dial(m.socket)
+		if err != nil {
+			return false, err
+		}
+		conn.Close()
+		return true, nil
+	}); err != nil {
 		return err
 	}
```

Comment:
returning `false, err` will fail the poll immediately and not fix the flake. this should look more like this:

```suggestion
	var lastDialErr error
	wait.PollImmediate(1*time.Second, 10*time.Second, func() (bool, error) {
		var conn *net.Conn
		_, conn, lastDialErr = dial(m.socket)
		if lastDialErr != nil {
			return false, nil
		}
		conn.Close()
		return true, nil
	})
	if lastDialErr != nil {
		return lastDialErr
	}
```

Reply:
Updated. Thanks.